### ISSUE TITLE
[client] header buttons (account, menu) should be more accessible

### ISSUE THREAD

**Issue by CodeMyst** [pastemyst/pastemyst-v3#4]
both buttons (which aren't even buttons now), should be interactive, focusable and have proper roles
